libraries/rush-lib/src/logic/DependencyAnalyzer.ts

```diff
@@ -104,6 +104,11 @@ export class DependencyAnalyzer {
             // For now, ignore local dependencies (that aren't cyclic dependencies).
             continue;
           }
+
+          if (localProject.splitWorkspace) {
+            // Ignore split workspace project
```

Comment:
Why are you skipping these?  For example we want `rush build`  to process split and un-split projects together -- only installation needs to care about the workspaces (if I understand correctly).

Reply:
Reverted.